Makefile

```diff
@@ -102,6 +102,14 @@ ifeq ($(COVERAGE), 1)
   LIB_LDFLAGS   += --coverage
 endif
 
+ifeq ($(ASAN), 1)
+  UFTRACE_CFLAGS    += -O0 -g -fsanitize=address
+  DEMANGLER_CFLAGS  += -O0 -g -fsanitize=address
+  SYMBOLS_CFLAGS    += -O0 -g -fsanitize=address
+  TRACEEVENT_CFLAGS += -O0 -g -fsanitize=address
+  TEST_CFLAGS       += -O0 -g -fsanitize=address
+endif
```

Comment:
I think you can just set `COMMON_CFLAGS` (and same for "TRACE" and "COVERAGE" too).

Reply:
Please see below:
- There's a conflict when preloading `libasan.so` and `libmcount.so` together if asan build is applied to `libmcount` so didn't use `COMMON_CFLAGS`.
- It's not also possible for "TRACE" because it applies to `libmcount*.so` that is not reenterant.
- `COMMON_CFLAGS` is already used for "COVERAGE".

Reply:
I ok then.